### DISCUSSION TITLE
Set boot_behavior

### DISCUSSION THREAD

**Discussion by cjaran** [gitbls/sdm#209] [General]
HI, I have a question, how can i set boot behavior to B4? I have it set in my burn to do boot_behavior, raspiconfig:boot_behaviour=B4, which is then invoked like this *sum([['--plugin', plugin] for plugin in plugins], []),

Here is the Logs: 
Mar 15 15:12:35 CAPSCREEN01 root[886]: sdm FirstBoot: Start Configuration with configuration file '/etc/sdm/auto-1piboot.conf'
Mar 15 15:12:35 CAPSCREEN01 root[897]: sdm FirstBoot: Set boot_behaviour 'B4'
Mar 15 15:12:39 CAPSCREEN01 root[1040]: sdm FirstBoot: Configuration with configuration file '/etc/sdm/auto-1piboot.conf' Completed
Mar 15 15:12:39 CAPSCREEN01 root[1067]: sdm FirstBoot: Set WiFi Country to 'US'
Mar 15 15:12:51 CAPSCREEN01 root[1665]: sdm FirstBoot: Waiting 10/120 seconds for system time synchronization
May 23 11:01:15 CAPSCREEN01 root[2123]: sdm FirstBoot: System time synchronization achieved
May 23 11:01:15 CAPSCREEN01 root[2132]: sdm FirstBoot: Run Generated sdm FirstBoot Script '/etc/sdm/0piboot/090-reset-quietness.sh'
May 23 11:01:15 CAPSCREEN01 root[2143]: sdm FirstBoot: Re-add 'quiet' to /boot/firmware/cmdline.txt
May 23 11:01:15 CAPSCREEN01 root[2146]: sdm FirstBoot: Re-add 'splash' to /boot/firmware/cmdline.txt
May 23 11:01:15 CAPSCREEN01 root[2153]: sdm FirstBoot: Run Custom sdm FirstBoot Script '/usr/local/sdm/1piboot/0-firstboot.sh'
May 23 11:06:30 CAPSCREEN01 root[11290]: sdm FirstBoot: Set boot behaviour to B3
May 23 11:06:33 CAPSCREEN01 root[11337]: sdm FirstBoot: Set systemd default to graphical.target
May 23 11:06:35 CAPSCREEN01 root[11357]: sdm FirstBoot: Start lightdm
May 23 11:06:35 CAPSCREEN01 root[11484]: sdm FirstBoot: Re-enable getty@tty1

**Comment by gitbls** [comment]
Please post the file `/etc/sdm/history` from the burned IMG (redacted if necessary). This will provide the details I need to sort out what's going on. Thx.
PS if you'd prefer to email rather than post it, my email is on the sdm README.
